src/Sylius/Bundle/ImportExportBundle/Form/Type/Reader/ImportReaderChoiceType.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ImportExportBundle\Form\Type\Reader;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
+
+/**
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+class ImportReaderChoiceType extends AbstractType
```

Comment:
It's the same as `ExportReaderChoiceType`. You can rename it to `reader_choice_type` and then declare twice in services using different alias for tag `form.type`.